optaplanner-examples/src/main/resources/org/optaplanner/examples/taskassigning_listvariable/solver/taskAssigningConstraints.drl

```diff
@@ -54,7 +54,7 @@ end
 
 rule "Minimze makespan (starting with the latest ending employee first)"
     when
-        Task(employee != null, nextTask == null, $endTime : endTime)
+        Employee(tasks != null, $endTime : endTime)
```

Comment:
If we're introducing this as a new example, maybe we don't do DRL anymore?
Not sure myself, just opening discussion.